packages/cape/src/websocket-connection.ts

```diff
@@ -71,7 +73,7 @@ export class WebsocketConnection {
    */
   connect(): Promise<void> {
     return new Promise((resolve, reject) => {
-      this.socket = new WebSocket(this.url);
+      this.socket = new WebSocket(this.url, '', this.options);
```

Comment:
I believe the protocol must be passed as an array of strings here, rather than setting the header: 

https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/WebSocket